### PR TITLE
Add 001 game creator

### PR DESCRIPTION
**Pull request by MichaelHinrichs**
### SteamDB app page links to a few games using this
https://steamdb.info/app/658890/
https://steamdb.info/app/731870/
https://steamdb.info/app/1222840/
https://steamdb.info/app/1272200/
https://steamdb.info/app/1511950/
https://steamdb.info/app/1551830/
https://steamdb.info/app/1572320/
https://steamdb.info/app/1574800/
https://steamdb.info/app/1886270/
https://steamdb.info/app/2105880/
https://steamdb.info/app/2107170/
https://steamdb.info/app/2442710/

### Brief explanation of the change
Find 001 game creator by it's 1RC file type.

### CODE REVIEW

**Comment by MichaelHinrichs** [comment]
Pull this.

**Comment by SteamTracker** [comment]
After running changed rules in this pull request *(this bot only tests rules.ini changes)*:

Changed technologies: **Engine.001GameCreator**

These apps now match: **(78)** [347400](https://steamdb.info/app/347400/depots/) [469980](https://steamdb.info/app/469980/depots/) [499960](https://steamdb.info/app/499960/depots/) [515880](https://steamdb.info/app/515880/depots/) [574810](https://steamdb.info/app/574810/depots/) [581000](https://steamdb.info/app/581000/depots/) [596690](https://steamdb.info/app/596690/depots/) [619520](https://steamdb.info/app/619520/depots/) [631530](https://steamdb.info/app/631530/depots/) [658890](https://steamdb.info/app/658890/depots/) [666910](https://steamdb.info/app/666910/depots/) [677770](https://steamdb.info/app/677770/depots/) [689220](https://steamdb.info/app/689220/depots/) [699570](https://steamdb.info/app/699570/depots/) [710520](https://steamdb.info/app/710520/depots/) [714900](https://steamdb.info/app/714900/depots/) [731870](https://steamdb.info/app/731870/depots/) [740100](https://steamdb.info/app/740100/depots/) [776240](https://steamdb.info/app/776240/depots/) [819130](https://steamdb.info/app/819130/depots/) [845480](https://steamdb.info/app/845480/depots/) [856460](https://steamdb.info/app/856460/depots/) [873120](https://steamdb.info/app/873120/depots/) [884170](https://steamdb.info/app/884170/depots/) [1054120](https://steamdb.info/app/1054120/depots/) [1082990](https://steamdb.info/app/1082990/depots/) [1111810](https://steamdb.info/app/1111810/depots/) [1125250](https://steamdb.info/app/1125250/depots/) [1140410](https://steamdb.info/app/1140410/depots/) [1175950](https://steamdb.info/app/1175950/depots/) [1181010](https://steamdb.info/app/1181010/depots/) [1222840](https://steamdb.info/app/1222840/depots/) [1225870](https://steamdb.info/app/1225870/depots/) [1233380](https://steamdb.info/app/1233380/depots/) [1265490](https://steamdb.info/app/1265490/depots/) [1267200](https://steamdb.info/app/1267200/depots/) [1272200](https://steamdb.info/app/1272200/depots/) [1274160](https://steamdb.info/app/1274160/depots/) [1284760](https://steamdb.info/app/1284760/depots/) [1335910](https://steamdb.info/app/1335910/depots/) [1351970](https://steamdb.info/app/1351970/depots/) [1392280](https://steamdb.info/app/1392280/depots/) [1422240](https://steamdb.info/app/1422240/depots/) [1470730](https://steamdb.info/app/1470730/depots/) [1477060](https://steamdb.info/app/1477060/depots/) [1511950](https://steamdb.info/app/1511950/depots/) [1542670](https://steamdb.info/app/1542670/depots/) [1551830](https://steamdb.info/app/1551830/depots/) [1565900](https://steamdb.info/app/1565900/depots/) [1572320](https://steamdb.info/app/1572320/depots/) [1574800](https://steamdb.info/app/1574800/depots/) [1669090](https://steamdb.info/app/1669090/depots/) [1690360](https://steamdb.info/app/1690360/depots/) [1761420](https://steamdb.info/app/1761420/depots/) [1782970](https://steamdb.info/app/1782970/depots/) [1798480](https://steamdb.info/app/1798480/depots/) [1826920](https://steamdb.info/app/1826920/depots/) [1879790](https://steamdb.info/app/1879790/depots/) [1881670](https://steamdb.info/app/1881670/depots/) [1886270](https://steamdb.info/app/1886270/depots/) [1914680](https://steamdb.info/app/1914680/depots/) [1944000](https://steamdb.info/app/1944000/depots/) [2013640](https://steamdb.info/app/2013640/depots/) [2013670](https://steamdb.info/app/2013670/depots/) [2105880](https://steamdb.info/app/2105880/depots/) [2107170](https://steamdb.info/app/2107170/depots/) [2112160](https://steamdb.info/app/2112160/depots/) [2180750](https://steamdb.info/app/2180750/depots/) [2319100](https://steamdb.info/app/2319100/depots/) [2442690](https://steamdb.info/app/2442690/depots/) [2442710](https://steamdb.info/app/2442710/depots/) [2458580](https://steamdb.info/app/2458580/depots/) [2494710](https://steamdb.info/app/2494710/depots/) [2523090](https://steamdb.info/app/2523090/depots/) [2639590](https://steamdb.info/app/2639590/depots/) [2652030](https://steamdb.info/app/2652030/depots/) [2692010](https://steamdb.info/app/2692010/depots/) [2837810](https://steamdb.info/app/2837810/depots/)

Please confirm that all these changed apps are correct.

File: rules.ini [hunk 1]
```diff
@@ -31,6 +31,9 @@ VSWAP = (?:^|/)vswap\. ; idTech0
 WAD = \.wad$ ; idTech1
 
 [Engine]
+001GameCreator[] = data\.1rc$
+001GameCreator[] = Math001\.dll$
+001GameCreator[] = Redist001\.exe$
```
Comment (Lifeismana) [inline rules.ini:36]:
```suggestion
001GameCreator = data\.1rc$
```

File: rules.ini [hunk 2]
```diff
@@ -31,6 +31,7 @@ VSWAP = (?:^|/)vswap\. ; idTech0
 WAD = \.wad$ ; idTech1
 
 [Engine]
+001GameCreator = ^data\.1rc$
```
Comment (Lifeismana) [inline rules.ini:34]:
```suggestion
001GameCreator = (?:^|/)data\.1rc$
```

**Comment by Lifeismana** [comment]
merged in 1b4529cf96d2821d1a1b47a214e0054f4defe1cf

**Pull request closed without merging**